packages/synapse-react-client/src/components/DownloadCart/DownloadListTable.tsx

```diff
@@ -228,6 +228,7 @@ export default function DownloadListTable(props: DownloadListTableProps) {
                 <th>
                   SynID
                   <InteractiveCopyIdsIcon
+                    size={'small'}
```

Comment:
InteractiveCopyIdsIcon renders a MUI [IconButton](https://mui.com/material-ui/react-button/#icon-button). The prop signature was changed in this PR to accept any IconButton props. In this case, it's appropriate to decrease the size.